ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/activity/PaywallActivity.kt

```diff
@@ -123,6 +176,26 @@ internal class PaywallActivity : ComponentActivity(), PaywallListener {
         setResult(RESULT_OK, createResultIntent(PaywallResult.Error(error)))
     }
 
+    private fun configureSdkWithSavedData(savedInstanceState: Bundle) {
+        val sdkConfigArgs = getSdkConfigArgs(savedInstanceState)
+        if (sdkConfigArgs == null) {
+            Logger.e("Missing SDK configuration arguments while restoring PaywallActivity")
+            return
+        }
+        Purchases.configure(
+            PurchasesConfiguration.Builder(this, sdkConfigArgs.apiKey)
+                .appUserID(sdkConfigArgs.appUserId)
+                .purchasesAreCompletedBy(sdkConfigArgs.purchasesAreCompletedBy)
+                .showInAppMessagesAutomatically(sdkConfigArgs.showInAppMessagesAutomatically)
+                .store(sdkConfigArgs.store)
+                .diagnosticsEnabled(sdkConfigArgs.diagnosticsEnabled)
+                .entitlementVerificationMode(sdkConfigArgs.verificationMode)
+                .dangerousSettings(sdkConfigArgs.dangerousSettings)
+                .pendingTransactionsForPrepaidPlansEnabled(sdkConfigArgs.pendingTransactionsForPrepaidPlansEnabled)
+                .build(),
+        )
```

Comment:
If we ever add a configuration option, we have to remember to add it to `SdkConfigArgs`. Can't really think of a solution though haha (other than some custom Detekt stuff or something).